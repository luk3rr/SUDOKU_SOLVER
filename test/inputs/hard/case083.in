905003004 300040509 600500200 802000600 000000020 000130005 057204306 060000700 000008000 
