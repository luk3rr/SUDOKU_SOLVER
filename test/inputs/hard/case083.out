Solving the following grid:
9 0 5 | 0 0 3 | 0 0 4 
3 0 0 | 0 4 0 | 5 0 9 
6 0 0 | 5 0 0 | 2 0 0 
------+-------+------
8 0 2 | 0 0 0 | 6 0 0 
0 0 0 | 0 0 0 | 0 2 0 
0 0 0 | 1 3 0 | 0 0 5 
------+-------+------
0 5 7 | 2 0 4 | 3 0 6 
0 6 0 | 0 0 0 | 7 0 0 
0 0 0 | 0 0 8 | 0 0 0 


Solution found :')

9 8 5 | 6 2 3 | 1 7 4 
3 2 1 | 8 4 7 | 5 6 9 
6 7 4 | 5 1 9 | 2 3 8 
------+-------+------
8 4 2 | 9 7 5 | 6 1 3 
5 1 3 | 4 8 6 | 9 2 7 
7 9 6 | 1 3 2 | 8 4 5 
------+-------+------
1 5 7 | 2 9 4 | 3 8 6 
4 6 8 | 3 5 1 | 7 9 2 
2 3 9 | 7 6 8 | 4 5 1 

Algorithm: BFS
Total time: 797 ms
Total expanded states: 106987
